### ISSUE TITLE
Comment on 2.2.6 Accessible Authentication

### ISSUE THREAD

**Issue by AidanA11y** [w3c/wcag21#473]
I think this is wholly impractical. It would make 2 factor authentication and One TIme Passcode mechanisms a failure. 

I do  appreciate  the barriers/challenges  and the need  for  accessible authentication  but this SC does not seem ready. It seems to be forcing content providers to choose between WCAG conformance and security of  the content.  

As I read this, asking for challenge questions (e.g your pet's name) would fail. This  is a common  technique which addresses the situation where a user cannot recall the password, but since it also requires recall it seems this useful technique to address password recall  would also fail. 

As written its not clear if "an authentication-credentials reset process" needs to be online of if a phone or in-person process would be sufficient. I suppose those could fail too as they may rely on recall, for example when an agent provides a temporary password.

**Comment by awkawk** [comment]
Official WG response: Thank you for your comment. This SC has been removed from the draft and will be able to be considered in a future revision.
